modules/virt-about-using-virtual-gpus.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-using-virtual-gpus_{context}"]
+= About using virtual GPUs with {VirtProductName}
+
+If you have a host with a compatible graphics processing unit (GPU), you can use it to create virtual GPUs (vGPUs). {VirtProductName} can automatically create vGPUs and other _mediated devices_ if an administrator provides configuration details in the `HyperConverged` custom resource (CR). This automation is especially useful for large clusters.
+
+[NOTE]
+====
+Refer to your hardware vendor's documentation for functionality and support details.
+====
+
+Mediated device:: A physical device that is divided into one or more virtual devices. A vGPU is a type of mediated device (mdev); the performance of the physical GPU is divided among the virtual devices. You can assign mediated devices to one or more virtual machines (VMs), though some GPUs do not support multiple guests.
```

Comment:
Request clarity here - if some GPUs do not support multiple guests, are we saying:
--you can assign the mediated devices anyway
or
--when you assign the mediated devices, there might be some failure because some GPUs do not support multiple guests.
Please clarify / restructure the statement.

Reply:
Ah, I can see how that would be confusing. I will rephrase it now.